Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -629,10 +611,8 @@ public final class ManifestLoader: ManifestLoaderProtocol {
                     return
                 }
 
-                // Wait to make sure we receive the last of the JSON output, and store it as the parsed manifest.
-                jsonOutputPipe.fileHandleForWriting.closeFile()
-                jsonOutputGroup.wait()
-                guard let jsonOutput = String(bytes: jsonOutputData, encoding: .utf8) else {
+                // Read the JSON output that was emitted by libPackageDescription.
+                guard let jsonOutput = try localFileSystem.readFileContents(jsonOutputFile).validDescription else {
```

Comment:
This restores the line of code to read the JSON file from before the manifest was compiled.